couchbase-sys/build.rs

```diff
@@ -50,17 +50,35 @@ fn main() {
         build_cfg.define("LIBCOUCHBASE_STATIC", "ON");
     }
 
+    if cfg!(any(target_os = "windows")) {
```

Comment:
I don't think you need the any with a single entry.. I think cfg!(target_os = "windows") should work?